dix/cm/st/dix_repair_st.sh

```diff
@@ -635,10 +635,10 @@ if [ ${#tests_list[@]} -eq 0 ]; then
 	declare -a tests_list=(
 				lookup
 				insert
-				get_during,
-				drop_during,
-				put_during,
-				del_repair,
+				get_during
+				drop_during
+				put_during
+				del_repair
 				del_rebalance
```

Comment:
Let's disable these tests by:
```
#                                get_during,
#                                drop_during
#                                put_during
#                                del_repair
#                                del_rebalance
```

Because the DIX repair is not yet fully tested.

Reply:
done